cli/js/lib.deno.ns.d.ts

```diff
@@ -2884,6 +2884,8 @@ declare namespace Deno {
     windowChange: () => SignalStream;
   };
 
+  /** Symbol to access Headers' cookies iterator */
+  export const cookiesIterator: unique symbol;
```

Comment:
I don't like that this is being exposed to the public types.

Reply:
I don't like it either, I thought it was needed, pretty new to TypeScript being honest, first time using it was here with Deno. Just removed it :)